missouri/model.yml

```diff
@@ -0,0 +1,98 @@
+model:
+  num_pop: 6000
+  time:
+    num_steps:
+    burn_steps:
+
+assort_mix:
+  assort_white:
+    attribute: race
+    agent_value: white
+    partner_values:
+      white: 0.5
+      black: 0.5
+  assort_black:
+    attribute: race
+    agent_value: black
+    partner_values:
+      white: 0.5
+      black: 0.5
+
+partnership:
+  sex:
+    frequency:
+      Sex: # is this correct bond type?
+        white: # race type?
```

Comment:
We don't actually have this broken down by race (because that can be a bit tricky when partners are different races). With NYC, Will and I had decided to take something like an average. The actual number of sex acts then gets calibrated through the race-specific condom usage.